### DISCUSSION TITLE
Is it possible to generate a single file for each change in release note?

### DISCUSSION THREAD

**Discussion by mahdiahmadirad** [rfennell/AzurePipelines#1555] [Q&A]
i want to show the release note according to deferences between the version already user installed and the release note of new release. so i need to have every change in a single entity(file) to enable the system to show only new changes in app.
i'm not sure that your extension designed for this solution. am i wright?

**Comment by rfennell** [comment]
The default is for the task finds all the changes (WI/CS) since the last successful run of the current pipeline.

You can override this behave and set a parameter for the pipeline run to compare against, that might help.

What we do is use the task to write a page in a WIKI, so each successful release adds a new page (or appends to a current page), thus building the list you are after

Have a look at https://www.youtube.com/watch?v=xaV3dFoQdV8&t=332s for ideas
